salt/_modules/metalk8s.py

```diff
@@ -691,3 +691,18 @@ def configure_archive(archive, remove=False):
     msg = "Archive '{0}' {1}".format(archive, msg)
     log.info(msg)
     return msg
+
+
+def backup_node():
+    metalk8s_version = __pillar__["metalk8s"]["cluster_version"]
+    backup_script = f"/srv/scality/metalk8s-{metalk8s_version}/backup.sh"
+    result = __salt__["cmd.run_all"](cmd=backup_script)
+    log.debug("Result: %r", result)
+
+    if result["retcode"] != 0:
+        output = result.get("stderr", result["stdout"])
```

Comment:
nitpick: I think stderr is always here but it's empty sometimes
```suggestion
        output = result.get("stderr") or result["stdout"]
```